.travis.yml

```diff
@@ -43,7 +43,7 @@ install:
       sudo apt-add-repository -y ppa:ubuntu-toolchain-r/test
       sudo apt-add-repository -y ppa:beineri/opt-qt562-trusty
       sudo apt-get -qq update
-      sudo apt-get -qq install g++-4.8 libc6-i386 qt56tools qt56svg qt56script qt56websockets
+      sudo apt-get -qq install g++-4.8 libc6-i386 qt56tools qt56svg qt56script qt56websockets libaspell-dev
```

Comment:
I'm very reluctant to now have an external dependency beside Qt to build at all 😞

Reply:
Well,
this is optional. So I can abandon this if you want. No problems.

Embedding aspell, is going to be very very difficult.

I chose external lib because, aspell is available everywhere (except windows)

Reply:
> Embedding aspell, is going to be very very difficult.

I wouldn't want to embed it neither for just having an optional 2nd spellchecker backend...
At best aspell support could be an optional flag that can be enabled at build time and if building doesn't take much longer this flag could be enabled by us in the different build systems.

Would users now also need to have the aspell installed to even run QOwnNotes?

Reply:
I have added the flags.

Let me test and confirm that it builds on mac/Linux with aspell.
Then I will add a global flag to turn it on/off completely.

> At best aspell support could be an optional flag that can be enabled at build time and if building doesn't take much longer this flag could be enabled by us in the different build systems.

Sounds good to me.